specifications.yml

```diff
@@ -436,14 +436,6 @@ components:
             - reminder
             - debt-collection
 
-    reminderId:
-      type: object
-      properties:
-            id:
-              type: string
-              description: The new reminder Id
-              example: "63b597c72e7e42972e5dec23"
-
     orderApplication:
```

Comment:
Goes back to the comment in the code ticket. you should just add more fields here, such as feeAMount and date :slightly_smiling_face: